test/factories/host_related.rb

```diff
@@ -1,13 +1,83 @@
 FactoryGirl.define do
+  factory :ptable do
+    sequence(:name) { |n| "ptable#{n}" }
+    layout 'zerombr yes\nclearpart --all    --initlabel\npart /boot --fstype ext3 --size=<%= 10 * 10 %> --asprimary\npart /     --f   stype ext3 --size=1024 --grow\npart swap  --recommended'
+  end
+
+  factory :parameter do
```

Comment:
I see proper spacing, so I'm going to assume @tstrachota fixed this and Github didn't notice...